src/aks-preview/azext_aks_preview/_help.py

```diff
@@ -178,6 +178,9 @@
         - name: --enable-private-cluster
           type: string
           short-summary: (PREVIEW) Enable private cluster.
+        - name: --enable-managed-identity
+          type: bool
+          short-summary: (PREVIEW) Using managed identity to manage cluster resource group.
```

Comment:
I suggest you call out the `system-assigned managed identity`. In future you can consider to expand the argument to take a list of used assigned ones

Reply:
Thanks for reminding, updated the help message.